src/EventStore.Core/TransactionLog/Chunks/TFChunkDb.cs

```diff
@@ -11,6 +11,7 @@ namespace EventStore.Core.TransactionLog.Chunks {
 	public class TFChunkDb : IDisposable {
 		public readonly TFChunkDbConfig Config;
 		public readonly TFChunkManager Manager;
+		public ITransactionFileTracker Tracker => _tracker;
```

Comment:
in the test that calls this you can probably just pass `new TFChunkTracker.NoOp()` instead

Reply:
fixed :+1: